### ISSUE TITLE
abstract keys away from user

### ISSUE THREAD

**Issue by trosel** [irislib/iris-angular#13]
Currently, the only way to login is to paste the whole private key into the login form.

https://github.com/irislib/iris-angular/blob/d83a3ca8da02f9470d96f979186677c7c04c50af/src/app/main/main.controller.coffee#L255

Users shouldn't have to touch their keys at all. Maybe try a password method. Or something like [Urbit ID](https://urbit.org/understanding-urbit/urbit-id/) where you have a "master ticket" that you memorize?

**Comment by amark** [comment]
oh, do we not use the GUN `User` username/password login prompt also? Notabug been using this for a long time now.

**Comment by trosel** [comment]
@amark No. Today, you have to copy and paste your private key in the "password" field in order to log in to a different computer 😁

**Comment by mmalmi** [comment]
Afaik, username/password method doesn't work offline. Also I don't like the possibility of brute-forcing accounts that have a bad password.

**Comment by trosel** [comment]
@mmalmi I wonder how Urbit does this with their ID and "master ticket". Their master ticket is only 4 6-letter words like `winter-paches-palfun-foslup`.

And you can log in anywhere with that.
